recipes/icu/all/conanfile.py

```diff
@@ -337,7 +340,9 @@ def package_info(self):
 
         if self.settings.os != "Windows" and self.options.data_packaging in ["files", "archive"]:
             data_path = os.path.join(self.package_folder, "res", self._data_filename).replace("\\", "/")
-            self.output.info("Appending ICU_DATA environment variable: {}".format(data_path))
+            self.output.info("Prepending to ICU_DATA runtime environment variable: {}".format(data_path))
+            self.runenv_info.prepend_path("ICU_DATA", data_path)
```

Comment:
I'm wondering whether it should be added to `self.buildenv_info` also when icu tools are enabled.